tests/test_vector_sqlite.py

```
@@ -0,0 +1,730 @@
+
+                # Verify database connection is open
+                assert adapter._db is not None
+
+                # Store reference to the database connection
+                db_connection = adapter._db
+
+            # After exiting the context manager, verify the connection is closed
+            try:
+                db_connection.execute("SELECT 1")
+                # If we get here, the connection is still open (which is unexpected)
+                assert False, "Database connection should be closed"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)